src/pages/home/HomePage.js

```diff
@@ -220,7 +220,7 @@ class App extends React.Component {
     render() {
         const hamburgerStyle = this.state.isHamburgerEnabled && this.props.isSidebarShown
             ? styles.hamburgerOpenAbsolute : styles.hamburgerOpen;
-        const visibility = !this.state.isHamburgerEnabled || this.props.isSidebarShown ? styles.dFlex : styles.dNone;
+        const visibility = !this.state.isHamburgerEnabled || this.props.isSidebarShown ? styles.w300 : styles.w0;
```

Comment:
It would be nice to prevent this bug from returning somehow. Can we please link to the issue you found in a doc here with a note that we are using `width` to toggle `visibility` because `display: 'none'` on the parent affects padding styles on the `TextInput`?